static/include/snippet/show-secret.md

```diff
@@ -3,7 +3,6 @@ By default, the sample code does not include your machine API key and API key ID
 We strongly recommend that you add your API key and API key ID as an environment variable and import this variable into your development environment as needed.
 
 To show your machine's API key and API key ID in the sample code, toggle **Include secret** on the **Code sample** tab.
-You can also see your API key and API key ID on your machine's **Security** tab.
```

Comment:
That's no longer true?

Reply:
No more security tab. I have another ticket to update that workflow